### PR TITLE
Update domains

### PR DESCRIPTION
**Pull request by Priyansxu**
Masduck discord bot developer

### CODE REVIEW

**Review by iostpa — CHANGES_REQUESTED** [review]
Friendly reminder that we still don't support NS records for the time being.

**Review by Priyansxu — COMMENTED** [review]
Updated record

File: domains/botlab.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "DBD(NO CODE)",
+  "owner": {
+    "username": "Priyansxu",
+    "email": "cloudypriyanshu@gmail.com"
+  },
+  "record": {
+    "A": "54.161.234.33"
```
Comment (CuteDog5695) [inline domains/botlab.json:8]:
```suggestion
    "A": ["54.161.234.33"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/botlab.json (hunk 1 above)
Comment (CuteDog5695) [inline domains/botlab.json:8]:
```suggestion
    "A": ["75.2.60.5"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Review by Priyansxu — COMMENTED** [review]
Mutedog

**Pull request merged**